mobile/src/libs/components/player/components/progress-bar/progress-bar.tsx

```diff
@@ -1,27 +1,34 @@
 import Slider from '@react-native-community/slider';
 import React from 'react';
+import { useColorScheme } from 'react-native';
```

Comment:
```suggestion
import { useColorScheme } from '#libs/hooks/hooks';
```

Reply:
we use it just for one time here

Reply:
Nevertheless all hooks should be exported from their file, even once)